src/js/components/Carousel/__tests__/Carousel-test.js

```diff
@@ -62,6 +62,37 @@ describe('Carousel', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('focus, blur and click events should be trigger', () => {
```

Comment:
Kind of gansta in your linguistications here, ya?

Reply:
Done.